src/index.html

```diff
@@ -0,0 +1,161 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Weather project</title>
+        <link rel="stylesheet"
+            href="https://fonts.googleapis.com/css?family=Ubuntu">
+
+        <link rel="stylesheet" type="text/css" href="index_style.css">
+
+    </head>
+    <body>
+        <div class="headerimg">
+            <img alt="leftcloudimg" class="leftcloud" src="../Assets/HTML&CSS/cloud-8121.png">
+            <img alt="rightcloudimg" class="rightcloud" src="../Assets/HTML&CSS/cloud-8121.png">
+            <div class="topsegment">
+                <div id="topfront">
+                    <div id="topimg1">
+                        <img id="topimg" src="../Assets/HTML&CSS/Icons_for_cities/maseru.svg" alt="x">
+                    </div>
+
+                    <div id="cityselect" style="position: relative;">
```

Comment:
```suggestion
                    <div id="cityselect">
```
Let's avoid using inline styling. We should prefer only to use styles via class names or id names through separate stylesheet